Library/Homebrew/cask/audit.rb

```diff
@@ -544,6 +539,13 @@ def audit_signing
             Please contact the upstream developer to let them know they should sign and notarize their software.
           EOS
         end
```

Comment:
```suggestion
          true
        end
```
I don't think we want to rely on the return value of `add_error` (since its signature says `void`).

Reply:
Yeah, this was an oversight on my part, was thinking how `odie` works 🤦‍♂️